tests/acceptance/features/transaction.feature

```diff
@@ -161,3 +161,8 @@ Feature: Transaction
       |  amount  | async  |      status     |
       |   1000   | true   |    processing   |
       |   1000   | false  | waiting_payment |
+
+  Scenario: Create a boleto transaction with a retrieved customer
+    Given a valid customer and retrieving the same customer
```

Comment:
@leonardo-barbosa correct me if I'm wrong but, for me, create a customer and retrieve it after doesn't make sense. 

My suggestion for this scenario is: 
```
Given an existent customer
When make a boleto transaction with random amount and metadata
Then the customer shouldn't be created again.
```

Makes sense?

Reply:
Yeah, it does make sense @leonampd .
I will change make this changes